pkg/controller/federatedcluster/clusterclient.go

```diff
@@ -101,3 +109,52 @@ func (self *ClusterClient) GetClusterHealthStatus() *fedv1a1.FederatedClusterSta
 
 	return &clusterStatus
 }
+
+// GetClusterZones gets the kubernetes cluster zones and region by inspecting labels on nodes in the cluster.
+func (self *ClusterClient) GetClusterZones() (zones []string, region string, err error) {
```

Comment:
(No action required) Why is zones a slice if only a single value is ever returned?